doc/userguide/upgrade.rst

```diff
@@ -82,6 +82,8 @@ Major changes
 - Unknown requirements in the ``requires`` keyword will now be treated
   as unmet requirements, causing the rule to not be loaded. See
   :ref:`keyword_requires`.
+- The following sticky buffers for matching FTP headers have been implemented:
```

Comment:
I still feel this doesn't belong here (and neither does the sip additions and the ldap addition)

Reply:
I'll remove all of them (separate commits for ldap/sip).

Do the new keywords need to be mentioned elsewhere?

Reply:
not outside of adding regular docs. Changelog will mention them as it lists the ticket subjects.